src/main/java/frc/robot/constants/AimingConstants.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.constants;
+
+import java.util.Map;
+
+import frc.robot.Util.PIDConstants;
+import frc.robot.subsystems.AimingSubsystem.AimingMotorMode;
+
+/** Add your docs here. */
+public class AimingConstants {
+
+
+    // Setpoints
+    //TODO: Find these
```

Comment:
Create an issue for this and paste the link to the issue in the comment.

Reply:
We have received information about setpoints from the design team.
The link to the issue: https://github.com/FRC-1294/frc2024/issues/262